### ISSUE TITLE
Choppa Morale-2 Tantrum Ability doesn't vanish after dead

### ISSUE THREAD

**Issue by wildwindblows** [WarEmu/WarBugs#20070]
**Expected behavior and actual behavior:**
Actual behavior: I used morale 2 Tantrum ability before i die and then after i released i see that Tantrum is still on. I really surprised to see that and I hit a mob after release and i saw that tantrum is still hitting. I still able to hit with tantrum after release.

Expected behavior: Correct me if im wrong but tantrum should be vanished after release because i already dead. Same issue is valid before and after rez too.

**Steps to reproduce the problem:**
1. Gain morale until morale-2 active
2. Use Tantrum ability> hit 1 or 2 time to mob> let mob kill you> Release> hit mob again after release and see tantrum is still hitting

**Testing Screenshots/Videos/Evidences (always needed):**
1. Before Dead: Pop Tantrum hit mob and dead.
[img]https://i.imgur.com/Ats8Kv6.jpg[/img]

2. After Release: See Tantrum is still on after dead and realease.
[img]https://i.imgur.com/Od0hjD3.jpg[/img]

3. Hit mobs after release to confirm it.
[img]https://i.imgur.com/1bIzeQi.jpg[/img]

Note: I hope you send me that nice looking manticore mount :)

Bests,
Sirzero.

**Comment by lilimx** [comment]
Hey,

hmmm that's a tricky question.....actually it should reset, but according this ticket #9724, it has its own weird mechanic (yeah).
I will lable it and a Dev should review it.

**Comment by wildwindblows** [comment]
Yes, I saw this post too before create post but i thought that post totally different from mine.

**Comment by wildwindblows** [comment]
FYI, bug is still valid before and after rez too. Tantrum is still active after rez.
